data-science-onramp/feature-engineering/feature_engineering.ipynb

```diff
@@ -1,4795 +1,1019 @@
 {
```

Comment:
<p>Original:</p><p>string, so it can be properly combined with the other datasets.</p><p> </p><p>Proposed:</p><p>string so <strong>that</strong> it can be properly combined with the other datasets.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahecole/python-docs-samples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Done</p>